telco-core/storage/odf-external/02-ocs-external-storagecluster.yaml

```diff
@@ -10,3 +10,6 @@ spec:
   externalStorage:
     enable: true
   labelSelector: {}
+status:
+  phase: Ready
```

Comment:
Is status really something we should be adding to the manifests?

Reply:
If/when deployed using Policy this status ensures that the CR has both been applied and has been successfully reconciled (taken effect). If the CRs are being directly applied your point is well taken, but when part of Policy they are critical. I considered putting them only in the reference PolicyGenTemplates but opted to include them here following the [pattern in RAN](https://github.com/openshift-kni/cnf-features-deploy/blob/master/ztp/source-crs/ClusterLogSubscription.yaml#L14-L15).